instrumentation/opentelemetry_phoenix/lib/opentelemetry_phoenix.ex

```diff
@@ -152,186 +132,72 @@ defmodule OpentelemetryPhoenix do
     :ok
   end
 
-  @doc false
-  def handle_endpoint_start(_event, _measurements, meta, config) do
-    Process.put({:otel_phoenix, :adapter}, config.adapter)
-
-    case adapter() do
-      :cowboy2 ->
-        cowboy2_start()
+  # TODO: do we still need exception handling? Only when cowboy?
```

Comment:
Don't we still want to have the `handle_router_dispatch_exception`?

Reply:
Did not appear so in testing.